src/main/java/com/minecolonies/coremod/items/ModItems.java

```diff
@@ -1,12 +1,12 @@
 package com.minecolonies.coremod.items;
 
-import com.minecolonies.coremod.blocks.*;
 import net.minecraft.item.Item;
 import net.minecraftforge.registries.IForgeRegistry;
 
 /**
  * Class handling the registering of the mod items.
  */
+@SuppressWarnings("squid:S1444")
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Suppressing the 'squid:S1444' warning is not allowed [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1309)